modules/internal/localeModels/flat/LocaleModel.chpl

```diff
@@ -230,7 +145,7 @@ module LocaleModel {
       return chpl_buildLocaleID(numLocales:chpl_nodeID_t, c_sublocid_none);
     }
     proc chpl_name() return local_name();
-    proc local_name() return "rootLocale";
+    proc local_name() return "rootLocale":string;
```

Comment:
That's odd - is the `:string` really necessary here or a leftover from an earlier time?